README.md

```diff
@@ -16,6 +16,7 @@ This code for sample application is intended for demonstration purposes only. It
 * kubectl is installed - https://docs.aws.amazon.com/eks/latest/userguide/install-kubectl.html
 * eksctl is installed - https://docs.aws.amazon.com/eks/latest/userguide/eksctl.html
 * jq is installed - https://jqlang.github.io/jq/download/
+* AWS CDK is installed - https://docs.aws.amazon.com/cdk/v2/guide/getting_started.html#getting_started_install
```

Comment:
Beside installation of AWS CDK with `npm install -g aws-cdk`, it also needs to run `cd cdk/ec2 && npm install` to fix following error.
```
TSError: ⨯ Unable to compile TypeScript:
bin/ec2.ts:3:22 - error TS2307: Cannot find module 'aws-cdk-lib' or its corresponding type declarations.

3 import * as cdk from 'aws-cdk-lib';
                       ~~~~~~~~~~~~~
bin/ec2.ts:20:10 - error TS2339: Property 'addDependency' does not exist on type 'IAMStack'.
```